docs/src/intro.md

```diff
@@ -78,5 +56,4 @@ export DATABASE_URL=postgresql://username:password@0.0.0.0:5439/postgis  # Conne
 .venv/bin/uvicorn eoapi.{SERVICE}.app:app --port 8000 --reload
 ```
 
-Note: services might have incompatible dependencies which you can resolve by using virtual environement per service
-
+Note: services might have incompatible dependencies which you can resolve by using a virtual environment for each service
```

Comment:
I'm confused by the overall change in this page. When I `refactored` it I tried to split the documentation from the eoapi demo application (hosted in eoapi repo) to just give an overview of what was the eoapi concept and why people should use it

Reply:
@vincentsarago should we just revert the intro to your previous version but keep the changes in `deployment.md`?

Reply:
done!